scripting/pugsetup.sp

```diff
@@ -2237,3 +2243,16 @@ stock bool PermissionFromString(const char[] permissionString, Permission& p,
 
   return true;
 }
+
+public void OnClientDisconnect(int client)
+{
+  if(GetClientCount() <= 2)
+	{
+		if (g_EndGameOnLastPlayerInServerCvar.BoolValue)
+		{
+			ServerCommand("sm_forceend");
+			ReadSetupOptions();
+			SetupFinished();
```

Comment:
What's the purpose of calling ReadSetupOptions & SetupFinished here?

Reply:
My bad.
I understand these functions wrong. Both removed.